src/main/java/com/google/api/gax/grpc/BundlingCallSettings.java

```diff
@@ -0,0 +1,90 @@
+package com.google.api.gax.grpc;
+
+import com.google.common.collect.ImmutableSet;
+
+import io.grpc.ManagedChannel;
+import io.grpc.MethodDescriptor;
+
+import java.io.IOException;
+import java.util.concurrent.ScheduledExecutorService;
+
+/**
+ * A settings class with generic typing that could used to configrue an bundling method
+ * or create the bundling callable object, which can be directly operated against an API.
```

Comment:
"which can be directly operated against an API" -> "which can make API method calls"